onmt/modules/copy_generator.py

```diff
@@ -1,16 +1,17 @@
 """ Generator module """
-import torch.nn as nn
 import torch
-import torch.cuda
+import torch.nn as nn
+import torch.nn.functional as F
```

Comment:
what is the rationale behind this ? see my comment below.

Reply:
You're correct, it should be `torch.sigmoid` instead of `F.sigmoid`. I prefer the function to the module because using the module requires defining an attribute on the `CopyGenerator` which is not strictly necessary. I prefer to only use `nn.Module` subclasses for things that have parameters. I forgot that `torch.nn.functional` was not the way to do that anymore.